### PR TITLE
Schedule detail view should default to first item in schedule [OC-116]

### PR DESCRIPTION
**Pull request by cameronblandford**
* Defaults the selected item to be whatever is first on the schedule. This can be changed to be the closest future event once logic for the related auto-scrolling is implemented.

### CODE REVIEW

File: app/components/meeting-schedule/component.js [hunk 1]
```diff
@@ -80,11 +81,5 @@ export default Ember.Component.extend({
...
         }
     }),
-    selectedItemId: Ember.computed(function () {
-        // return this.get('model.items').then((results) => {
-        //     console.log(results.get('firstObject.id'));
-        //     return results.get('firstObject.id');
-        // });
-        return 2;
-    })
+    selectedItemId: 0
```
Comment (cameronblandford) [inline app/components/meeting-schedule/component.js]: Worth noting that this creates a 404 request to the API whenever the page is loaded but then rectifies itself almost instantaneously and makes the proper GET call.

File: app/components/meeting-schedule/component.js [hunk 2]
```diff
@@ -47,6 +47,7 @@ export default Ember.Component.extend({
                     retList.push([i]);
                 }
             });
+            this.set('selectedItemId', retList[0][0].id);
```
Comment (samanehsan) [inline app/components/meeting-schedule/component.js]: What happens if there are no items in `retList`?

Comment (cameronblandford) [inline app/components/meeting-schedule/component.js]: Then you shouldn't be using the schedule component ;) just kidding, I'll add a check for that!

**Comment by samanehsan** [comment]
Looks good to me! :octocat:

**Pull request merged**